.github/actions/on_device_tests/action.yaml

```diff
@@ -74,6 +74,7 @@ runs:
             ${LOADER_PLATFORM:+"--loader_platform" "$LOADER_PLATFORM"} \
             ${DIMENSION:+"--dimension" "$DIMENSION"} \
             ${USE_SHARDING:+"--unittest_shard_index" "${{ matrix.shard }}"} \
+            ${TEST_ATTEMPTS:-"--test_attempts" "$TEST_ATTEMPTS"} \
```

Comment:
I think this should be ${TEST_ATTEMPTS:+"--test_attempts" "$TEST_ATTEMPTS"} vs ${TEST_ATTEMPTS:-"--test_attempts" "$TEST_ATTEMPTS"} ?

Reply:
Yes, I think you are right. I tested this a bit more locally and found that it should work even if the variable isset to empty string, which I think is the case if the field is not in the platform config.

I also renamed the variable to be clear that it's specific to on device tests.